<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reactor</class>
 <widget class="QDialog" name="Reactor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reactor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QGroupBox" name="groupFuel">
     <property name="title">
      <string>Fuel Burnup</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineFuelBurnup">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupThermal">
     <property name="title">
      <string>Thermal Power Correction</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineThermal"/>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="buttonThermalOff">
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Off</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="buttonThermalOn">
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>On</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupAuto">
     <property name="title">
      <string>Automatic Reactor Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="8">
       <widget class="QPushButton" name="buttonAutoOn">
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>On</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QLineEdit" name="lineControlError"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="buttonAutoOff">
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Off</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="buttonSetpointInc">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="labelFlux">
        <property name="text">
         <string>Neutron Flux</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="labelControlError">
        <property name="text">
         <string>Control Error</string>
        </property>
       </widget>
      </item>
      <item row="2" column="9">
       <widget class="QLabel" name="labelFluxLog">
        <property name="text">
         <string>Neutron Flux Log</string>
        </property>
       </widget>
      </item>
      <item row="3" column="7" rowspan="2">
       <widget class="QLineEdit" name="linePowerSetpoint"/>
      </item>
      <item row="1" column="9">
       <widget class="QLineEdit" name="lineFlux"/>
      </item>
      <item row="3" column="9" rowspan="2">
       <widget class="QLineEdit" name="lineFluxLog"/>
      </item>
      <item row="2" column="7">
       <widget class="QLabel" name="labelPowerSetpoint">
        <property name="text">
         <string>Power Setpoint</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="buttonSetpointDec">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="8" rowspan="2">
       <widget class="QLineEdit" name="lineNeutron"/>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="buttonSetpointDecMore">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>--</string>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QLabel" name="labelNeutron">
        <property name="text">
         <string>Neutron Rate</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="buttonSetpointIncMore">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>++</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="groupScram">
     <property name="title">
      <string>Reactor Scram</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QPushButton" name="buttonScramAuto">
        <property name="text">
         <string>Auto</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="buttonScramReset">
        <property name="text">
         <string>Reset</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="buttonScramTrip">
        <property name="text">
         <string>Trip</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
